LivingCity/traffic/b18CommandLineVersion.cpp

```diff
@@ -153,10 +155,19 @@ void B18CommandLineVersion::runB18Simulation() {
     infile.close();
```

Comment:
Might need to update partition file
`vertexIdToPar` finally leads to segmentation fault in `b18InitCUDA_n`